src/Compilers/CSharp/Portable/Symbols/Source/LambdaSymbol.cs

```diff
@@ -416,6 +416,8 @@ internal override bool GenerateDebugInfo
 
         internal override bool IsInitOnly => false;
 
+        internal override bool IsCallerUnsafe => false;
```

Comment:
It looks like `unsafe` is disallowed on lambdas, but I can't find a test capturing that. Consider including one.
Spec: should we add such support?